File: BOJ/10409_1.cpp
// 서버
// https://www.acmicpc.net/problem/10409
#include <iostream>
using namespace std;
int N, T;
int main()
{
    ios::sync_with_stdio(0);
    cin.tie(0);
    cout.tie(0);

    cin >> N >> T;
    int total = 0, cnt = 0;
    for (int i = 0; i < N; i++)
    {
        int num;
        cin >> num;
        total += num;

        if (total <= T)
            cnt++;
    }

    cout << cnt;
    return 0;
}

File: BOJ/10409_1.py
'''
    서버
    https://www.acmicpc.net/problem/10409
'''
N, T = map(int, input().split())
work = list(map(int, input().split()))
total = 0
cnt = 0
for i in range(len(work)):
    total += work[i]
    if(total > T):
        break
    cnt += 1
print(cnt)

'''
    참고
    https://dojang.io/mod/page/view.php?id=2179

    m=lambda:map(int,input().split())
    _,t=m()
    s=0
    j=0
    for i in m():
        if s+i>t:break
        j+=1;s+=i
    print(j)
'''
